### ISSUE TITLE
About overlap

### ISSUE THREAD

**Issue by wuchunxu123** [pallashadow/SuperGlue-pytorch#9]
Hi, thanks for your excellent work，it's really impressive. I have a question. It is mentioned that the author selects image pairs having overlap from 40% to 80% for training. Is the overlap value used to balance the numbers of matched points and unmatched points? I noticed that the pos part and neg part share the same weight in the loss function. If it is, can I use the matching percentage of keypoints in Scannet to represent the overlap and generate image pairs？I think this is more efficient and can directly balances the number of matched and unmatched points. Looking forward to your reply.

**Comment by pallashadow** [comment]
去superglue的这些变种项目里看吧，我已经不研究这个了
https://zhuanlan.zhihu.com/p/387962855

**Comment by wuchunxu123** [comment]
好的，多谢
